packages/dev/storybook-builder-parcel/index.js

```diff
@@ -0,0 +1,239 @@
+const {Parcel} = require('@parcel/core');
+const path = require('path');
+const {createProxyMiddleware} = require('http-proxy-middleware');
+const {normalizeStories, loadPreviewOrConfigFile} = require('@storybook/core-common');
+const fs = require('fs');
+
+exports.start = async function ({options, router}) {
+  let parcel = await createParcel(options, true);
+
+  router.use(async (req, res, next) => {
+    if (req.url === '/' || req.url === '/index.html') {
+      return next();
+    }
+
+    let proxy = createProxyMiddleware({
+      target: 'http://localhost:3000/',
```

Comment:
any reason we aren't setting it to run on 9003 like it has been?
looks like chromatic storybook still runs on 9004?

Reply:
Storybook's server is still on 9003. This is proxying the parcel dev server.

Reply:
the cli does print out 3000, doesn't really matter though

Reply:
yeah... well parcel prints that. Storybook still prints 9003 also. Kinda confusing...